code/modules/reagents/reagent_containers/blood_pack.dm

```diff
@@ -163,10 +163,10 @@
 /obj/item/reagent_containers/blood/attackby(obj/item/attacking_item, mob/user)
 	..()
 	if (attacking_item.ispen())
-		if (REAGENT_VOLUME(reagents, /singleton/reagent/blood) && name != "empty blood pack") //Stops people mucking with bloodpacks that are filled
+		if (REAGENT_VOLUME(reagents, /singleton/reagent/blood) && name != "empty IV bag") //Stops people mucking with bloodpacks that are filled
```

Comment:
Making blood bag data into a singleton would avoid this really ugly name check, lol